src/jace/util/compiling.py

```diff
@@ -0,0 +1,146 @@
+# JaCe - JAX Just-In-Time compilation using DaCe (Data Centric Parallel Programming)
```

Comment:
`compiling` is a misleading name for this module, as even the docstring states that it does more than just compilation. I'm also not sure if this module belongs to `.util` and not as regular top-level module. Why not just rename to something like `jace.execution`?

Reply:
Execution is misleading in the same way as it does more than just execute the SDFG.

Reply:
I have now moved them to `dace_helper.py`.